samcli/cli/main.py

```diff
@@ -97,6 +97,8 @@ def cli(ctx):
     sam_cli_logger = logging.getLogger("samcli")
     sam_cli_formatter = logging.Formatter("%(message)s")
     lambda_builders_logger = logging.getLogger("aws_lambda_builders")
+    botocore_logger = logging.getLogger("botocore")
 
     SamCliLogger.configure_logger(sam_cli_logger, sam_cli_formatter, logging.INFO)
     SamCliLogger.configure_logger(lambda_builders_logger, sam_cli_formatter, logging.INFO)
+    SamCliLogger.configure_null_logger(botocore_logger)
```

Comment:
Does this silence all logs, including debug logs?

Reply:
https://docs.python.org/3.6/howto/logging.html#configuring-logging-for-a-library explains it. Basically it will prevent the library’s logged event being output to sys.stderr and is documented as "A do-nothing handler". This is also what boto3 is doing but botocore is not (for whatever reason).

From reading the docs, my understanding is yes.